packages/gp2-components/src/organisms/__tests__/OnboardingPageFooter.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+import OnboardingPageFooter from '../OnboardingPageFooter';
+
+describe('OnboardingPageFooter', () => {
+  const defaultProps = {
+    isContinueEnabled: false,
+    publishUser: jest.fn(),
+  };
+  it('renders the links with the right props', () => {
+    render(
+      <OnboardingPageFooter
+        {...defaultProps}
+        previousHref="/previous"
+        continueHref="/next"
+        isContinueEnabled
+      />,
+      {
+        wrapper: MemoryRouter,
+      },
+    );
+    const links = screen.getAllByRole('link') as HTMLAnchorElement[];
+    expect(links.map(({ textContent }) => textContent)).toMatchObject([
+      expect.stringMatching('Sign Out'),
+      expect.stringMatching('Previous'),
+      expect.stringMatching('Continue'),
+    ]);
+    expect(links.map(({ href }) => href)).toMatchObject([
+      expect.stringMatching('/logout'),
+      expect.stringMatching('/previous'),
+      expect.stringMatching('/next'),
+    ]);
+  });
+  it('renders publish button if theres on continueHref', () => {
+    render(
+      <OnboardingPageFooter
+        {...defaultProps}
+        continueHref={undefined}
+        isContinueEnabled
+      />,
+      {
+        wrapper: MemoryRouter,
+      },
+    );
+    expect(
+      screen.queryByRole('link', { name: 'Continue' }),
+    ).not.toBeInTheDocument();
+    expect(screen.getByRole('button', { name: 'Publish' })).toBeVisible();
+  });
+  it('calls the publishUser function when pressing the publish button', async () => {
```

Comment:
I don't think we need the async. But I think the linting should have highlighted this?